client/src/components/Comments/index.tsx

```diff
@@ -38,47 +40,57 @@ class Comments extends React.Component<ICommentsProps> {
         }
     }
 
-    public componentWillUnmount(): void {
-        this.props.resetComments();
-    }
-
     public getMoreComments = (): void => {
-        if (!this.props.allCommentsLoaded && this.props.postId) {
-            this.props.getComments(this.props.postId, this.props.commentsPage + 1);
+
+        if (!!this.props.postId) {
+            const commentStateForCurrentPost =
+                this.props.commentsPage.filter((info: { postId: string, page: number }) =>
+                    info.postId === this.props.postId);
+            // @ts-ignore
+            if (!this.props.allCommentsLoaded.includes(this.props.postId) && commentStateForCurrentPost.length) {
+                // @ts-ignore
+                return this.props.getComments(this.props.postId, commentStateForCurrentPost[0].page + 1);
+            }
+            this.props.showAlert('All comments loaded', 'warning');
```

Comment:
do we nned this warning on UI?

Reply:
yes
when server doesn't have any comments more same alert will work in action, but only at first time
all next cases it will work here